online_portfolio/static/portfolio/js/edit.js

```diff
@@ -651,8 +650,18 @@ function updateSkillName(ele, event) {
 
 }
 
-function skillRemove(i_index, j_index) {
-    console.log(i_index);
-    console.log(j_index);
-    console.log(skills);
-}
\ No newline at end of file
+function setDeleteButtonpath(path) {
```

Comment:
@HarshitaAgarwal09 I discovered that there is no need to do this, just doing `const DeleteButtonPath = "<django url>"` in the portfolio.html file `<script>` tag is enough. Please change it to that method.

Reply:
@HarshitaAgarwal09 since it was the only change, I fixed it on your behalf.